rollup.config.mjs

```diff
@@ -22,14 +22,15 @@ export default {
       entries: [
         { find: '@hanafi', replacement: path.resolve(__dirname, 'contents/hanafi/') },
         { find: '@components', replacement: path.resolve(__dirname, 'src/components/') },
-        { find: '@utils', replacement: path.resolve(__dirname, 'src/utils/') }
+        { find: '@utils', replacement: path.resolve(__dirname, 'src/utils/') },
+        { find: '@src', replacement: path.resolve(__dirname, 'src/') },
       ]
     }),
     markdownContentPlugin(),
     json(),
     postcss({
       extensions: ['.css'],
-      modules: true
+      modules: false // to prevent changing the classNames while building
```

Comment:
Brother, this is a very fundamental change. CSS modules ensure that styles are modular, e.g. not global. Historically speaking, global state has always given issues where a single change ends up affecting many different places unintentionally because everything is global